rskj-core/src/main/java/org/ethereum/config/blockchain/testnet/TestNetDifficultyDropEnabledConfig.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ * (derived from ethereumJ library, Copyright (c) 2016 <ether.camp>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.ethereum.config.blockchain.testnet;
+
+import co.rsk.core.BlockDifficulty;
+import org.ethereum.core.BlockHeader;
+
+public class TestNetDifficultyDropEnabledConfig extends TestNetAfterBridgeSyncConfig {
+
+    @Override
+    public BlockDifficulty calcDifficulty(BlockHeader currentBlockHeader, BlockHeader parentBlockHeader) {
+        // If more than 10 minutes, reset to minimum difficulty
+        if (currentBlockHeader.getTimestamp() >= parentBlockHeader.getTimestamp() + 600) {
```

Comment:
Could you move 600 to a variable. Is it in seconds right ?

Reply:
Done!